tests/snuba/api/endpoints/test_organization_events.py

```diff
@@ -0,0 +1,15 @@
+from tests.snuba.api.endpoints.test_organization_events_v2 import (
+    OrganizationEventsV2EndpointTest,
+    OrganizationEventsV2MetricsEnhancedPerformanceEndpointTest,
+)
+
+
+class OrganizationEventsEndpointTest(OrganizationEventsV2EndpointTest):
+    viewname = "sentry-api-0-organization-events"
+    referrer = "api.organization-events"
+
+
+class OrganizationEventsMetricsEnhancedPerformanceEndpointTest(
+    OrganizationEventsV2MetricsEnhancedPerformanceEndpointTest
+):
+    viewname = "sentry-api-0-organization-events"
```

Comment:
nit: but lets keep this in the same file as events_v2, so later we can `git mv` and keep history